python/cuml/test/test_fil.py

```diff
@@ -492,29 +492,39 @@ def test_output_args(small_classifier_and_preds):
 
 
 def to_categorical(features, n_categorical):
+    # returns data in two formats: pandas (for LightGBM) and numpy (for FIL)
     # the main bottleneck (>80%) of to_categorical() is the pandas operations
     n_features = features.shape[1]
     df_cols = {}
     # all categorical columns
     cat_cols = features[:, :n_categorical]
-    cat_cols = cat_cols - cat_cols.min(axis=1, keepdims=True)  # range [0, ?]
-    cat_cols /= cat_cols.max(axis=1, keepdims=True)  # range [0, 1]
+    cat_cols = cat_cols - cat_cols.min(axis=0, keepdims=True)  # range [0, ?]
```

Comment:
Why does the axis index change?

Reply:
This clearly requires a documentation comment: which axis corresponds to rows, which to columns, and what does the `axis` parameter specify, i.e. the axis which is reduced or the one which remains. A link to numpy documentation would be useful.